src/lib.rs

```diff
@@ -173,6 +173,30 @@ pub struct SimplificationOptions {
     pub join_nested_linear_containers: bool,
 }
 
+impl SimplificationOptions {
+    /// Make a [`SimplificationOptions`] with all simplifications turned off.
+    ///
+    /// This makes it easy to run a single simplification type on a tree:
+    /// ```
+    /// # use egui_tiles::*;
+    /// # let mut tree: Tree<()> = Tree::empty("tree");
+    /// tree.simplify(&SimplificationOptions {
+    ///     prune_empty_tabs: true,
+    ///     ..SimplificationOptions::off()
+    /// });
+    ///
+    pub fn off() -> Self {
+        Self {
```

Comment:
This should be a `pub const OFF: Self` instead!

Reply:
makes sense 👍🏻